MekHQ/src/mekhq/campaign/personnel/Bloodname.java

```diff
@@ -64,7 +64,16 @@
 	public static final int P_PROTOMECH = 4;
 	public static final int P_NAVAL = 5;
 	public static final int P_NUM = 6;
-
+/*
```

Comment:
Is this to help remember these values?

Reply:
This is for my bloodname merge, as we've currently got two different lists, one in person and one in Bloodname. I was going to do it now, but wanted to cut down on the scope creep a bit. I've added a TODO message to remind myself.